oss-internship-2020/pffft/README.md

```diff
@@ -38,22 +47,22 @@ it is taken into account while testing.
 In the end, the performance of PFFFT library it is outlined by the output.*
 
 #### CMake observations resume:
-    * linking pffft and fftpack (which contains necessary functions for pffft)
-    * set math library 
+* linking pffft and fftpack (which contains necessary functions for pffft)
+* set math library 
 
 #### Sandboxed main observations resume:
-    * containing two testing parts (fft / pffft benchmarks)
-    * showing the performance of the transformations implies 
+* containing two testing parts (fft / pffft benchmarks)
+* showing the performance of the transformations implies 
     testing them through various FFT dimenstions. 
     Variable N, the input length, will take specific values 
     meaning the number of points to which it is set the calculus 
     (more details of mathematical purpose of N - https://en.wikipedia.org/wiki/Cooley%E2%80%93Tukey_FFT_algorithm). 
-    * output shows speed depending on the input length
+* output shows speed depending on the input length
 
     
 
 ### Bugs history
-    - [Solved] pffft benchmark bug: "Sandbox not active"  
+    1. [Solved] pffft benchmark bug: "Sandbox not active"  
```

Comment:
The ordinals should not be indented in Markdown. That is, the following will render in monospace:
```
  1. First
     Continuation of first...
  2. Second
^^ Spaces
```

Whereas this will render correctly as a list:
```
1. First
   Continuation of first...
2. Second
```

yielding

1. First
   Continuation of first...
2. Second